nx/include/switch/services/set.h

```diff
@@ -262,6 +262,128 @@ typedef struct {
     u32 extra_color;      ///< Extra Color.
 } SetSysHomeMenuScheme;
 
+typedef struct {
+    u8 offset[0x6];
+} SetCalAccelerometerOffset;
+
+typedef struct {
+    u8 scale[0x6];
+} SetCalAccelerometerScale;
+
+typedef struct {
+    u8 cert[0x70];
+} SetCalAmiiboEcdsaCertificate;
+
+typedef struct {
+    u8 cert[0x20];
+} SetCalAmiiboEcqvBlsCertificate;
+
+typedef struct {
+    u8 key[0x44];
+} SetCalAmiiboEcqvBlsKey;
+
+typedef struct {
+    u8 cert[0x90];
+} SetCalAmiiboEcqvBlsRootCertificate;
+
+typedef struct {
+    u8 cert[0x14];
+} SetCalAmiiboEcqvCertificate;
+
+typedef struct {
+    u8 key[0x54];
+} SetCalAmiiboKey;
+
+typedef struct {
+    u8 calibration[0x9];
+} SetCalAnalogStickFactoryCalibration;
+
+typedef struct {
+    u8 parameter[0x12];
+} SetCalAnalogStickModelParameter;
+
+typedef struct {
+    u8 battery_lot[0x18];
+} SetCalBatteryLot;
+
+typedef struct {
+    u8 bd_addr[0x6];
+} SetCalBdAddress;
+
+typedef struct {
+    u8 cfgId1[0x1E];
+} SetCalConfigurationId1;
+
+typedef struct {
+    u8 offset[0x6];
+} SetCalConsoleSixAxisSensorHorizontalOffset;
+
+typedef struct {
+    u32 code[0x4];
+} SetCalCountryCode;
+
+typedef struct {
+    u32 offset;                     ///< Relative to current position.
+    u8 cert[0x17C];
+} SetCalEccB233DeviceCertificate;
+
+typedef struct {
+    u8 key[0x58];
+} SetCalEccB233DeviceKey;
+
+typedef struct {
+    u8 cert[0x400];
+} SetCalGameCardCertificate;
+
+typedef struct {
+    u32 size;           ///< Size of the entire key.
+    u8 key[0x130];
+} SetCalGameCardKey;
+
+typedef struct {
+    u8 offset[0x6];
+} SetCalGyroscopeOffset;
+
+typedef struct {
+    u8 scale[0x6];
+} SetCalGyroscopeScale;
+
+typedef struct {
+    u8 addr[0x6];
+} SetCalMacAddress;
+
+typedef struct {
+    u32 offset;                     ///< Relative to current position.
+    u8 cert[0x23C];
+} SetCalRsa2048DeviceCertificate;
+
+typedef struct {
+    size_t size;            ///< Size of the entire key.
```

Comment:
If this is 32bit use u32, same below (otherwise use u64).

Reply:
oops! Good catch.

Reply:
I have fixed this.